course04/homework00/ciaranfrain/sorted_array.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <deque>
+
+class SortedArray
+{
+public:
+	SortedArray(std::size_t maximumSize);
+	
+	void clear() { mStorage.clear(); }
+	const int& operator[](int n) const { return Get(n); }
```

Comment:
we discussed the rule about what to chose between value and const ref for function parameters, and the same applies to return type: this should return by value, as `int` is small.